packages/server/tests/utils.ts

```diff
@@ -306,6 +306,24 @@ export default class TestUtils {
         this.logger.info(`Balances of the new account: ${balance.toString()}`);
     };
 
+    deployContract = async(contract, client: Client, privateKey) => {
+        const fileCreateTx = new FileCreateTransaction()
```

Comment:
nit: Meant to double check but i think the SDK has a `ContractCreateFLow` (or something similar) that might do all this for us.
Try it out once, if it doesn't work immediately with like 5 mins trying then revert and leave as is :)

Reply:
This function will be refactored when [PR 211](https://github.com/hashgraph/hedera-json-rpc-relay/pull/211) gets merged. I would prefer to try it then.

Reply:
Works for me